CHANGELOG.md

```diff
@@ -7,6 +7,8 @@ Mapbox welcomes participation and contributions from everyone.
 
 #### Bug fixes and improvements
 - Fixed reroute request interruption when setting the `NavigationRerouteController` [#5950](https://github.com/mapbox/mapbox-navigation-android/pull/5950).
+- Introduced `RoutesSetCallback` parameter to `MapboxNavigation#setNavigationRoutes`, which is called after the routes passed to `MapboxNavigation#setNavigationRoutes` are processed or are failed to be processed [#5945](https://github.com/mapbox/mapbox-navigation-android/pull/5945).
```

Comment:
Maybe?
```suggestion
- Introduced `RoutesSetCallback` parameter to `MapboxNavigation#setNavigationRoutes`. The callback is called after new routes are processed or are failed to be processed [#5945](https://github.com/mapbox/mapbox-navigation-android/pull/5945).
```

Reply:
Why do you want to remove the specification of the routes ("passed to `MapboxNavigation#setNavigationRoutes`")? "New routes" sounds vague.